editor.planx.uk/src/@planx/components/FindProperty/Public/index.tsx

```diff
@@ -89,21 +90,6 @@ function Component(props: Props) {
     },
   );
 
-  // if allowNewAddresses is on, fetch the boundary geojson for this team to position the map view or default to London
-  //   example value for team.settings.boundary is https://www.planning.data.gov.uk/entity/8600093.geojson
-  const { data: geojson } = useSWR(
-    () =>
-      props.allowNewAddresses && teamSettings?.boundary
-        ? teamSettings.boundary
-        : null,
-    fetcher,
-    {
-      shouldRetryOnError: true,
-      errorRetryInterval: 500,
-      errorRetryCount: 1,
-    },
-  );
```

Comment:
We don't actually need to make this fetch, we're getting the `boundaryBBox` directly from the Store in the map in the child component !